index.js

```diff
@@ -1,33 +1,48 @@
-var http, director, cool, bot, router, server, port;
+// Set the bot token from MeBots in .env under BOT_TOKEN
+let mebots = require('mebots');
+let bot = new mebots.Bot('your_bot_shortname_here', process.env.BOT_TOKEN);
 
-http        = require('http');
-director    = require('director');
-cool        = require('cool-ascii-faces');
-bot         = require('./bot.js');
+// Setup our web routing
+const express = require("express");
+const app = express();
 
-router = new director.http.Router({
-    '/' : {
-        post: bot.respond,
-        get: ping
-    }
+// Library to send POST requests simply
+const axios = require('axios')
```

Comment:
I am hesitant to complicate this example with an arbitrary library like this; I feel that we should stick to the most basic implementation wherever possible so consider maybe not using axios

Reply:
this is the most basic implementation. without using axios or a similar library, we get code like this:
```
const https = require('https');

function httpsPost({body, ...options}) {
    return new Promise((resolve,reject) => {
        const req = https.request({
            method: 'POST',
            ...options,
        }, res => {
            const chunks = [];
            res.on('data', data => chunks.push(data))
            res.on('end', () => {
                let body = Buffer.concat(chunks);
                switch(res.headers['content-type']) {
                    case 'application/json':
                        body = JSON.parse(body);
                        break;
                }
                resolve(body)
            })
        })
        req.on('error',reject);
        if(body) {
            req.write(body);
        }
        req.end();
    })
}
```